src/librbd/io/ImageDispatcher.h

```diff
@@ -33,7 +33,7 @@ class ImageDispatcher : public Dispatcher<ImageCtxT, ImageDispatcherInterface> {
   void shut_down(Context* on_finish) override;
 
   void apply_qos_schedule_tick_min(uint64_t tick) override;
-  void apply_qos_limit(uint64_t flag, uint64_t limit, uint64_t burst) override;
+  void apply_qos_limit(uint64_t flag, uint64_t limit, uint64_t burst, uint64_t burst_seconds) override;
```

Comment:
Nit: try to avoid lines over 80 or so characters in length. Drop `uint64_t burst_seconds` down to the next line (indented)